### PR TITLE
Rails 3.1.3 nominally functional

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/controllers/system_status_controller.rb [hunk 1]
```diff
@@ -99,20 +88,20 @@ def details
...
+      revision: Git.revision,
+      branch: Git.branch,
       hostname: `hostname`.chomp,
       cache: cache_message,
       user_count_positive: User.all.any?,
       registered_deployment_id: Rails.cache.read('registered-deployment-id'),
       environment_deployment_id: ENV['DEPLOYMENT_ID'],
       last_migration: {
-        cas: cas,
+        cas:,
```
Comment (yakloinsteak) [inline app/controllers/system_status_controller.rb]: Something's missing.

File: app/controllers/system_status_controller.rb [hunk 2]
```diff
@@ -85,8 +74,8 @@ def details
       end
 
       jobs_stats[queue] = {
-        enqueued: enqueued,
-        failed: failed,
+        enqueued:,
```
Comment (yakloinsteak) [inline app/controllers/system_status_controller.rb]: Something's missing?

File: app/controllers/system_status_controller.rb [hunk 3]
```diff
@@ -72,7 +61,7 @@ def details
     jobs_stats = {}
     jobs_message = 'OK'
     Delayed::Job.select('distinct queue').pluck(:queue).each do |queue|
-      scope = Delayed::Job.where(queue: queue)
+      scope = Delayed::Job.where(queue:)
```
Comment (yakloinsteak) [inline app/controllers/system_status_controller.rb]: Something's missing

**Review by yakloinsteak — CHANGES_REQUESTED** [review]
I think there was a copy/paste error in the system status controller that needs fixing. Looks good otherwise.

File: app/controllers/system_status_controller.rb (hunk 1 above)
Comment (eanders) [reply app/controllers/system_status_controller.rb]: You'd think.  That's ruby's new hotness in 3.1.  I'm not 100% sold on it, but rubocop is. https://blog.saeloun.com/2021/09/28/ruby-allow-value-omission-in-hash-literals.html

Comment (yakloinsteak) [reply app/controllers/system_status_controller.rb]: My professional opinion on that syntax is that it stinks. It's like `<blink>` or `goto`. I'm 0% sold on it :-)

Comment (eanders) [reply app/controllers/system_status_controller.rb]: Agreed.  Updating .rubocop.yml now

**Pull request merged**